Codeinwp/neve
# Seamless View Cart Button styles fixed

### Summary

Before the PR, view cart button (that occurs after the add to cart button clicked) where in the shop page was not acting like secondary button.

With this PR, view cart button looks like secondary button and it can be set by customizer -> buttons -> secondary buttons.

### Will affect visual aspect of the product

YES

### Screenshots 

### Test instructions

1. Install Neve, Neve Pro, WooCommerce
2. enable seamless button on archive (on WooCommerce Settings -> Products -> Enable AJAX add to cart buttons on archives)
3. enable seamless button on single product (on Customizer -> WooCommerce -> Single Product -> Enable Seamless Add to Cart)
4. change secondary button style on Customizer -> Buttons -> Secondary Button
5. both view cart button should looks like a secondary. (you should able to control view cart button styles on customizer -> buttons -> secondary buttons.)


Closes codeinwp/neve-pro-addon#1210.
